docs/Show.md

```diff
@@ -198,6 +142,140 @@ const Aside = () => {
+
+React-admin provides 2 built-in show layout components:
+
+- [`<SimpleShowLayout>`](./SimpleShowLayout.md) displays fields with a label in a single column
+- [`<TabbedShowLayout>`](./TabbedShowLayout.md) displays a list of tabs, each tab rendering a stack of fields with a label
+
+To use an alternative layout, switch the `<Show>` child component:
+
+```diff
+export const PostShow = () => (
+    <Show>
+-       <SimpleShowLayout>
++       <TabbedShowLayout>
++           <TabbedShowLayout.Tab label="Main">
+                <TextField source="title" />
+                <TextField source="teaser" />
+                <RichTextField source="body" />
+                <DateField label="Publication date" source="created_at" />
++           </TabbedShowLayout.Tab>
+-       </SimpleShowLayout>
++       </TabbedShowLayout>
+    </Show>
+);
+```
+
+You can also pass a React element as child, to build a custom layout. Check [Building a custom Show Layout](./ShowTutorial.md#building-a-custom-layout) for more details.
+
+**Tip**: Use [`<ShowGuesser>`](./ShowGuesser.md) instead of `<Show>` to let react-admin guess the fields to display based on the dataProvider response.
+
+## `component`
+
+By default, the Show view renders the main content area inside a Material UI `<Card>`. The actual layout of the record fields depends on the Show Layout component you're using (`<SimpleShowLayout>`, `<TabbedShowLayout>`, or a custom layout component).
+
+You can override the main area container by passing a `component` prop:
+
+{% raw %}
+```jsx
+import { Box } from '@mui/material';
+
+const ShowWrapper = ({ children }) => (
+    <Box sx={{ margin: 2, border: 'solid 1px grey' }}>
+        {children}
+    </Box>
+);
+
+// use a ShowWrapper as root component
+const PostShow = props => (
+    <Show component={ShowWrapper} {...props}>
+        ...
+    </Show>
+);
+```
+{% endraw %}
+
+## `disableAuthentication`
+
+By default, the `<Show>` component will automatically redirect the user to the login page if the user is not authenticated. If you want to disable this behavior and allow anonymous access to a show page, set the `disableAuthentication` prop to `true`.
+
+```jsx
+const PostShow = () => (
+    <Show disableAuthentication>
+        ...
+    </Show>
+);
+```
+
+## `emptyWhileLoading`
+
+By default, `<Show>` renders its child component even before the `dataProvider.getOne()` call returns. If you use `<SimpleShowLayout>` or `<TabbedShowLayout>`, this isn't a problem as these components only render when the record has been fetched. 
+
+But if you use a custom child component that expects the record context to be defined, your component will throw an error. For instance, the following will fail on load with a "ReferenceError: data is not defined" error:
+
+```jsx
+import { Show, useShowContext } from 'react-admin';
+import { Stack, Typography } from '@mui/icons-material/Star';
+
+const SimpleBookShow = () => {
+    const { data } = useShowContext();
```

Comment:
```suggestion
    const { record } = useShowContext();
```

my understanding is that `data` is deprecated, and we should use `record` instead. ([source](https://github.com/marmelab/react-admin/blob/master/packages/ra-core/src/controller/show/useShowController.ts#L122))

Same applies later in the same file.